internal/cmd/destinations.go

```diff
@@ -5,6 +5,7 @@ import (
 	"fmt"
 
 	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
```

Comment:
Oops, I missed this on earlier review. This should be `gopkg.in/yaml.v2`, which is the more standard yaml library. 


```suggestion
	"gopkg.in/yaml.v2"
```